python/GafferUITest/BoolWidgetTest.py

```diff
@@ -102,5 +102,31 @@ def testDisplayMode( self ) :
 		w = GafferUI.BoolWidget( displayMode = GafferUI.BoolWidget.DisplayMode.Switch )
 		self.assertEqual( w.getDisplayMode(), w.DisplayMode.Switch )
 
+	def testIndeterminateState( self ) :
+
+		w = GafferUI.BoolWidget()
+
+		w.setState( w.State.Indeterminate )
+		self.assertEqual( w.getState(), w.State.Indeterminate )
+
+		w.setState( False )
+		self.assertIs( w.getState(), False )
+
+		w.setState( True )
+		self.assertIs( w.getState(), True )
+
+		# We don't want users to be able to set the
+		# indeterminate state. It is a display-only
+		# value.
+
+		w._qtWidget().click()
+		self.assertIs( w.getState(), False )
+
+		w._qtWidget().click()
+		self.assertIs( w.getState(), True )
+
+		w._qtWidget().click()
+		self.assertIs( w.getState(), False )
+
```

Comment:
Added in 4aa9bb5. I've just discovered the `git commit --fixup` option, which formats the commit message so that a later `git rebase --autosquash` will squash it into the correct commit. So I'm going to give that a go for all the fixups here.

Reply:
Cool, if this works - will do the same.